### PR TITLE
software-stack/lab: Explain the write() system call

### LINKED ISSUE #115: software-stack/lab: Replace or explain the `write()` syscall
The `write()` syscall is used multiple times in the Software Stack lab [1, 2, 3]. This is problematic because the students don't know what a file descriptor is at this point in the course. We need to add some lines [here](https://github.com/open-education-hub/operating-systems/blob/master/content/chapters/software-stack/lab/content/basic-syscall.md) that **briefly** explain that file descriptors will be studied later in the course and that for now, it suffices to know what 0, 1 and 2 are.

[1] https://github.com/open-education-hub/operating-systems/blob/master/content/chapters/software-stack/lab/support/basic-syscall/hello.asm
[2] https://github.com/open-education-hub/operating-systems/blob/master/content/chapters/software-stack/lab/support/syscall-wrapper/syscall.s
[3] https://github.com/open-education-hub/operating-systems/blob/master/content/chapters/software-stack/lab/support/common-functions/syscall.s

### PR DESCRIPTION
**Pull request by andreia-oca**
Add a paragraph to explain the `write()` syscall. Briefly mention file descriptors.

### CODE REVIEW

**Review by Alex-deVis — APPROVED** [review]

File: content/chapters/software-stack/lab/content/basic-syscall.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ Hello, world!
...
 The two programs end up printing the `Hello, world!` message at standard output by issuing the `write` system call.
 Then they complete their work by issuing the `exit` system call.
+
+The `write` system call writes a buffer to the file referred by the first argument which is the file descriptor.
+File descriptors are going to be studied in-depth in future chapters.
+For now, it is enough for you to know that:
+
+* `0` references the standard input (`stdin`)
+* `1` references the standard output (`stdout`)
+* `2` references the standard error (`stderr`)
```
Comment (teodutu) [inline content/chapters/software-stack/lab/content/basic-syscall.md:47]:
```suggestion
- `0` references the standard input (`stdin`)

- `1` references the standard output (`stdout`)

- `2` references the standard error (`stderr`)
```

File: content/chapters/software-stack/lab/content/basic-syscall.md [hunk 2]
```diff
@@ -37,6 +37,15 @@ Hello, world!
 
 The two programs end up printing the `Hello, world!` message at standard output by issuing the `write` system call.
 Then they complete their work by issuing the `exit` system call.
+
+The `write` system call writes a buffer to the file referred by the first argument which is the file descriptor.
+File descriptors are going to be studied in-depth in future chapters.
+For now, it is enough for you to know that:
```
Comment (teodutu) [inline content/chapters/software-stack/lab/content/basic-syscall.md:43]:
```suggestion
For now, it is enough for you to know that they are integers that behave like file handlers.
The 3 most common file descriptors are: 
```

**Review by teodutu — CHANGES_REQUESTED** [review]
See my suggestions and then we can merge this.

**Review by teodutu — APPROVED** [review]
I'm OK with this. Thanks!

**Review by Alex-deVis — APPROVED** [review]
If @teodutu approved it what else can I say?

**Pull request merged**